src/v0/destinations/google_cloud_function/transform.js

```diff
@@ -1,29 +1,19 @@
 const _ = require('lodash');
 const {
-  defaultRequestConfig,
-  defaultPostRequestConfig,
   getSuccessRespEvents,
   checkInvalidRtTfEvents,
   handleRtTfSingleEventError,
 } = require('../../util');
 
-const { generateBatchedPayload, validateDestinationConfig, addHeader } = require('./util');
+const { generateBatchedPayload, validateDestinationConfig } = require('./util');
 
 // Main process Function to handle transformation
 function process(event) {
   const { message, destination } = event;
-  const { googleCloudFunctionUrl } = destination.Config;
-
   // Config Validation
   validateDestinationConfig(destination);
 
-  const response = defaultRequestConfig();
-  // adding header
-  addHeader(response, destination.Config);
-  response.method = defaultPostRequestConfig.requestMethod;
-  response.body.JSON = message;
-  response.endpoint = googleCloudFunctionUrl;
-
+  const response = message;
```

Comment:
Effectively there's no transformation happening here - only task is to segregate the successful vs failed ones in case of same event getting retried ?

Reply:
Yes, just some validations and batching.